config.xml

```diff
@@ -92,17 +92,15 @@
   <plugin name="ionic-plugin-keyboard" spec="^2.2.1" />
   <plugin name="cordova-plugin-local-notification" spec="^0.9.0-beta.3" />
   <plugin name="cordova-plugin-app-version" spec="^0.1.9" />
-  <plugin name="cordova-plugin-fcm" spec="git+https://github.com/yatharthranjan/cordova-plugin-fcm.git" />
   <plugin name="cordova-plugin-globalization" spec="^1.11.0" />
-  <plugin name="cordova-plugin-firebase-analytics" spec="^1.0.1">
-    <variable name="FIREBASE_CORE_VERSION" value="16.0.+" />
-  </plugin>
   <plugin name="phonegap-plugin-mobile-accessibility" spec="^1.0.5" />
   <plugin name="cordova-media-with-compression" spec="^2.1.2" />
   <plugin name="cordova-plugin-file" spec="^6.0.1" />
   <plugin name="cordova-plugin-insomnia" spec="^4.3.0" />
+  <plugin name="cordova-plugin-advanced-http" spec="^2.0.9" />
```

Comment:
Why exactly are we moving to `cordova-http`? What additional feature does this HTTP have? I would assume Angular `HttpClient` is a fairly popular and standard one. I am in favor of using the angular features and keeping the plugin dependency to the minimal. 
Plus, does this break the support with running the app on browser? in that case, I would strongly advice reconsidering this option, for the sake of development and maintenance.

Reply:
I was getting HTTP errors with the IOS build with the `cordova-plugin-firebase` installed. It got fixed by using the http plugin. Let me check again.

Reply:
Sorry for some reason the error is not there anymore and it works with `cordova-plugin-firebase` so `cordova-http` is not needed. I have made the changes.